Assets/Scripts/Utility/MathUtils.cs

```diff
@@ -0,0 +1,38 @@
+using System.Linq;
+using System.Collections.Generic;
+
+namespace Utility
```

Comment:
tut mir leid aber diese .cs verwirrt mich

Reply:
Sie implementiert zwei Funktionen, die mehrfach aufgerufen werden und in eine eigene statische Klasse gehören. Übersetzt sich zu "Math utilities"